docsrc/conf.py

```diff
@@ -301,22 +301,22 @@
 for tuple in pathset:
     os.chdir(tuple[0])
     for rstfile in glob.glob("*.rst"):
-        author = [("The Cyrus Team")]
```

Comment:
Brief explanation because the need for this change might not be obvious: In Python, for loops do not affect variable scoping. This causes the author variable to bleed into the global scope, where it becomes the author configuration option. However that option needs to be a string while this variable is a list, and this causes Sphinx 2 to fail because it tries to perform a character translation on the list.

Reply:
Thanks!